src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -142,6 +162,8 @@ public static class EditPersonDescriptor {
         private Email email;
         private RoomNumber roomNumber;
         private Address address;
+        private Name emergencyName;
+        private Phone emergencyPhone;
```

Comment:
Should these be emergencyContactName and emergencyContactPhone to make it clearer?

Reply:
No, I don't think so.
I used emergencyName and emergencyPhone to be consistent with the parameter name in the feature specification (EMERGENCY_NAME and EMERGENCY_PHONE).